test/specs/example.e2e.ts

```diff
@@ -0,0 +1,23 @@
+describe('WDIO VSCode Service', function () {
+  this.retries(3);
```

Comment:
I think that we can do something like `before(() => browser.pause(3000));`

Reply:
That worked! Thank you 🙇‍♀️